index.js

```diff
@@ -78,7 +78,13 @@ class NgRename {
       return dir
     }
     catch (e) {
-      return this.findNgProjectDir(`${dir}/..`)
+      try {
+        fs.readFileSync(`${dir}/angular.json`) // for angular 6
```

Comment:
I don't think nested try blocks are a good solution.

How about:

      const angular_bigger_than6 = `${dir}/angular.json`
      const angular_lower_than6 = `${dir}/.angular-cli.json`
      if (fs.existsSync(angular_bigger_than6)) {
        fs.readFileSync(angular_bigger_than6)
        return dir
      } else if (fs.existsSync(angular_lower_than6)) {
        fs.readFileSync(angular_lower_than6)
        return dir
      } else {
        return this.findNgProjectDir(`${dir}/..`)
      }